src/components/CompanyProfile/index.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+import { Form, Col, Row, Container, Button, Image } from 'react-bootstrap'
+import uploadImage from '../../media/upload-Image.png'
+const CompanyProfile = () => {
+  return (
+    <Container fluid="md">
+      <h2 className="row justify-content-center">Company Profile</h2>
+      <Row>
+        <Col xs>
+          <Form>
+            <Form.Group
+              className="form-group-set"
+              as={Row}
+              controlId="formElement"
+            >
+              <Form.Label column sm="4">
+                Company Name
+              </Form.Label>
+              <Col sm="8">
+                <Form.Control
+                  className="text-field"
+                  type="text"
+                  placeholder="Comapny Name"
+                />
+              </Col>
+            </Form.Group>
+
+            <Form.Group
+              className="form-group-set"
+              as={Row}
+              controlId="formElement"
+            >
+              <Form.Label column sm="4">
+                Address
```

Comment:
This components doesn't exist in the develop branch. We have a CompanyProfile in the pages folder and it doesn't look like this.